src/modules/licence-import/load/connectors/queries.js

```diff
@@ -135,5 +135,69 @@ exports.createLicence = `
     expired_date=excluded.expired_date,
     lapsed_date=excluded.lapsed_date,
     revoked_date=excluded.revoked_date,
-    date_updated=now();
+    date_updated=now()
+  returning licence_id;
+`;
+
+exports.createLicenceVersion = `
+  insert into water.licence_versions (
+    licence_id,
+    issue,
+    increment,
+    status,
+    start_date,
+    end_date,
+    external_id,
+    date_created,
+    date_updated
+  )
+  values ($1, $2, $3, $4, $5, $6, $7, now(), now())
+  on conflict (external_id) do update
+  set
+    status = excluded.status,
+    start_date = excluded.start_date,
+    end_date = excluded.end_date,
+    date_updated = now()
+  returning licence_version_id;
+`;
+
+exports.createLicenceVersionPurpose = `
+  insert into water.licence_version_purposes (
+    licence_version_id,
+    purpose_primary_id,
+    purpose_secondary_id,
+    purpose_use_id,
+    abstraction_period_start_day, abstraction_period_start_month,
+    abstraction_period_end_day, abstraction_period_end_month,
+    time_limited_start_date, time_limited_end_date,
+    notes, annual_quantity,
+    date_created, date_updated
+  ) values (
+    $1,
+    (select purpose_primary_id from water.purposes_primary where legacy_id = $2),
+    (select purpose_secondary_id from water.purposes_secondary where legacy_id = $3),
+    (select purpose_use_id from water.purposes_uses where legacy_id = $4),
+    $5, $6,
+    $7, $8,
+    $9, $10,
+    $11, $12,
+    now(), now()
+  )
+  on conflict (
```

Comment:
Not certain this on conflict spec is valid - I think there can be multiple licence version purposes with the same purpose/abs period.